src/Blocks/custom/form-selector/form-selector-editor.scss

```diff
@@ -0,0 +1,37 @@
+// stylelint-disable declaration-no-important
+
+.es-block-form-selector {
+	&__title,
+	&__title-after {
+		text-align: center;
+		font-style: italic;
+		font-size: 14px;
```

Comment:
Doesn't need to be base-font-size calc'd, but would be nice if it was in `rem`s 😄